.github/workflows/cd.yml

```diff
@@ -74,6 +77,7 @@ jobs:
         env:
           OLD_VERSION: ${{ env.OLD_VERSION }}
           VERSION: ${{ steps.get-version.outputs.version }}
+          GH_TOKEN: ${{ steps.retrieve-secrets.outputs.github-pat-bitwarden-devops-bot-repo-scope }}
```

Comment:
A `GH_TOKEN` env variable is not needed to commit if you check out with `token` param. Example here: https://github.com/bitwarden/sdk/blob/c04b9a0e950fc79bd14464156030a169f0db14ef/.github/workflows/release-swift.yml#L206-L211